### ISSUE TITLE
@slidy/svelte: A11y: <button> cannot have role 'group'

### ISSUE THREAD

**Issue by Valexr** [Valexr/Slidy#87]
```
[WARNING] [plugin esbuild-svelte] A11y: <button> cannot have role 'group'
│ 5   src/components/Thumbnail/Thumbnail.svelte:46:2:
│ 6     46 │ 44:   {#each slides as item, i (item.id ?? getImgSrc(item) ?? i)}
│ 7        ╵   ^
│ 85:     {@const title = i18n.slideN.replace("%s", (i + 1).toString())}
│ 96:     <button
│20       ^
│ 17:       aria-current={i === active ? "true" : undefined}
│ 28:       aria-label={title}
```
